### PR TITLE
zh-trans website/content/zh/docs/tasks/administer-cluster/guaranteed-…

### PR DESCRIPTION
**Pull request by zhangx501**
…scheduling-critical-addon-pods.md

根据英文版本删减，重新更新了中文译文，同时更改了部分译文，让阅读更流畅

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md [hunk 1]
```diff
@@ -1,54 +1,46 @@
 ---
-approvers:
 - davidopp
 - filipg
 - piosz
 title: 关键插件 Pod 的调度保证
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md]: 请附上英文，方便对照。

File: content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md [hunk 2]
```diff
@@ -1,54 +1,46 @@
...
+content_template: templates/concept
 ---
 
-{{< toc >}}
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md]: {{< toc >}}
英文原文有吗？为什么要把他删除？

File: content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md [hunk 3]
```diff
@@ -1,54 +1,46 @@
...
+{{% capture body %}}
 
-## 二次调度器：关键插件的调度保证
 
-二次调度器确保关键的插件总是能被调度（假定普通的 Pod 不存在时，集群具有足够的资源去运行关键插件 Pod）。
-如果调度器确定没有节点有足够资源去运行关键插件 Pod，假使有 Pod 已经运行在集群中（通过将关键插件 Pod 的条件 PodScheduled 设置为 false，原因设置为 Unschedulable  来指示），这时二次调度器通过清理一些 Pod 尽量去释放空间，然后调度器将调度该插件 Pod。
+<!--
+### Marking pod as critical
+-->
+### 将pod标记为	`critical`
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md]:
```suggestion
### 将pod标记为 `critical`
```

File: content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md [hunk 4]
```diff
@@ -1,54 +1,46 @@
...
 
+<!--
+To be considered critical, the pod has to run in the `kube-system` namespace (configurable via flag) and
+->
+要将pod标记为	`critical` ，pod 必须在kube-system命名空间中运行（可通过标志配置），以及
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md]:
```suggestion
要将pod标记为 `critical`，pod 必须在 kube-system 命名空间中运行（可通过标志配置），以及
```

**Comment by xichengliudui** [comment]
@zhangx501 ,请问源码的地址在哪里？

File: content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md [hunk 5]
```diff
@@ -1,54 +1,46 @@
...
 
+<!-- 
+In addition to Kubernetes core components like api-server, scheduler, controller-manager running on a master machine
+there are a number of add-ons which, for various reasons, must run on a regular cluster node (rather than the Kubernetes master).
+Some of these add-ons are critical to a fully functional cluster, such as metrics-server, DNS, and UI.
+A cluster may stop working properly if a critical add-on is evicted (either manually or as a side effect of another operation like upgrade)
+and becomes pending (for example when the cluster is highly utilized and either there are other pending pods that schedule into the space
+vacated by the evicted critical add-on pod or the amount of resources available on the node changed for some other reason).
+-->
+除了在主机上运行的Kubernetes核心组件（如  api-server 、scheduler 、controller-manager ）之外，还有许多插件，由于各种原因，
```
Comment (markthink) [inline content/zh/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md]: 注意中英文间要留空格

**Pull request merged**